CHANGELOG.md

```diff
@@ -210,6 +210,7 @@ users about errors and where they originate.
 - Cursors in text editors behave correctly now (they are not affected by scene
   pan and zoom). This was possible because of the new multi-camera management
   system implemented in EnsoGL.
+- [Fixed method names highlighted in pink][1408]
```

Comment:
1. Not a valid sentence.
2. This should be better explained to the users, like "There was a bug introduced after one of the latest Engine updates, that sent ... types, which made .. pink. This is fixed now".

Reply:
But the "Unresolved" type was never shown to Users?